### ISSUE TITLE
Add Python 3 support

### ISSUE THREAD

**Issue by nehemiah-hivery** [sebdah/dynamodb-config-store#21]
Installing on Python 3.6.5 gives ConfigParser error which is removed in Python 3, so we need to use either configparser2 or a back-ported configparser. 

```
()~> python --version
Python 3.6.5
() pip install dynamodb-config-store

Collecting dynamodb-config-store
  Using cached https://files.pythonhosted.org/packages/45/b0/ee8344f8d4f6a5592a94d0240aa733a2570bdc7cc97260a71f4d220b4ea5/dynamodb-config-store-0.2.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/kw/k_sd90fd56v_ppm425b0rp9h0000gn/T/pip-install-f826pw4m/dynamodb-config-store/setup.py", line 4, in <module>
        from ConfigParser import SafeConfigParser
    ModuleNotFoundError: No module named 'ConfigParser'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/kw/k_sd90fd56v_ppm425b0rp9h0000gn/T/pip-install-f826pw4m/dynamodb-config-store/
```

**Comment by sebdah** [comment]
Hey!
Happy to see someone using this little lib :smile: and thanks for your email :bowing_man:.

I am not actively using it in any project at the moment. I noticed that you forked the lib. If you feel like taking a stab at making it Python 3 compatible that would be :100: :taco: and :tada:. I'd be more than happy to review and merge.

**Comment by nehemiah-hivery** [comment]
Thx for the message, Here is the PR

https://github.com/sebdah/dynamodb-config-store/pull/22

**Comment by sebdah** [comment]
@nehemiah-hivery thank you for your contribution on this. I have just now released version 0.3.0 to PyPI.

:bowing_man:
